### PR TITLE
JNG-4852 Test static primitive query dao functionality

### PR DESCRIPTION
**Pull request by gaborflorian**
<table>
<td>
  <a href="https://blackbelt.atlassian.net/browse/JNG-4852" title="JNG-4852" target="_blank">TaskJNG-4852</a>  Test static primitive query dao functionality
  </td></table>
  <br />

### CODE REVIEW

File: judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/TestStaticQueryDao.java [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+        assertEquals(LocalTime.of(22, 59, 59), timeWithoutParamQueryDao.execute());
+        assertEquals(
+                LocalTime.of(12, 24, 45),
+                timeWithParamQueryDao.execute(TimeWithParamQueryParameter.builder().withTimeHour(12).withTimeMinute(24).withTimeSecond(45).build())
+        );
+        assertEquals(
+                LocalTime.of(13, 4, 5),
+                timeWithDefaultParamQueryDao.execute(TimeWithDefaultParamQueryParameter.builder().build())
+        );
+
```
Comment (bencegelei) [inline judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/TestStaticQueryDao.java:209]: Timestamp is missing (adding it should be an "almost" copy-paste)

File: judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/entity/AssociationRelationshipsTest.java [hunk 2]
```diff
@@ -183,13 +183,7 @@ public void testTraverse() {
...
     })
     public void testDeletingRequiredRelationThrowsException() {
-        IllegalStateException thrown = assertThrows(
-                IllegalStateException.class,
-                () -> entityCDao.delete(entityC)
-        );
 
-        assertTrue(thrown.getMessage().contains("There is mandatory references which is not removable"));
-        assertTrue(thrown.getMessage().contains("#singleRequiredConA"));
     }
```
Comment (bencegelei) [inline judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/entity/AssociationRelationshipsTest.java]: Why was this deleted?

Comment (gaborflorian) [reply judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/entity/AssociationRelationshipsTest.java]: Fix it

Comment (gaborflorian) [reply judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/entity/AssociationRelationshipsTest.java]: [1a37435](https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/pull/175/commits/1a37435b20fb4d2843ff2f08ba668cdab23b636b)

File: judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/TestStaticQueryDao.java (hunk 1 above)
Comment (gaborflorian) [reply judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/TestStaticQueryDao.java:209]: [1a37435](https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/pull/175/commits/1a37435b20fb4d2843ff2f08ba668cdab23b636b)

**Review by bencegelei — APPROVED** [review]

**Review by noherczeg — APPROVED** [review]
